course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -29,12 +32,19 @@ print(f"{name} is a friend")
 - [What is an if statement?](https://python.tecladocode.com/2_countries_visited/2_if_statements.html#what-is-an-if-statement)
 
 ### The compound `if` statement - `elif` keyword
-- `elif` keyword is used for chaining multiple if statements
-- `elif` block will be entered only if the previous `if`/`elif` block doesn't get entered
-- the syntax is the same as with writing the `if` statement, the difference being that the `elif` keyword is used instead of `if`
-- `elif` keyword should be indented the same number of times as the previous `if`/`elif` keyword
+- `elif` keyword is used for chaining multiple `if` statements
+- if the last `if`/`elif` block fails, the next `elif` will be checked, etc.
+- syntax for using the `elif` keyword:
```

Comment:
I think the 4 lines here could be reduced by making use of the title to introduce the section. Then we could reword the second bullet point slightly. I think the third bullet point wouldn't be necessary. What do you think of this?

### The `elif` keyword

- Used for chaining multiple `if` statements
- Only one of the branch bodies can run: the first one to evaluate to `True`

```py
if <expr>:
    <first_statement>
elif <expr>:
    <second_statement>
```